core/src/subgraph/context.rs

```diff
@@ -64,7 +58,7 @@ where
     C: Blockchain,
 {
     instance: SubgraphInstance<C, T>,
-    pub instances: SubgraphKeepAlive,
+    pub instances: Arc<SubgraphKeepAlive>,
```

Comment:
What's the rationale for removing `CheapClone` from `SubgraphKeepAlive` and wrapping it in `Arc`? Is it because now it has two `Arc` fields?

Reply:
oh it's because cheap_clone is already implemented for Arc and I've now wrapped with Arc. I don't think it's actually copied so I could probably leave it without the Arc anyway

Reply:
I think wrapping everything with `Arc` hinders readability, so I try to avoid it unless some fields are quite expensive to clone. We're good though, feel free to merge.

Reply:
I think it's a good point about readability, I've reverted that change.

Reply:
💯